ui/DesignSystem/Tooltip.svelte

```diff
@@ -6,19 +6,21 @@
  LICENSE file.
 -->
 <script lang="typescript">
-  import { CSSPosition } from "ui/src/style";
-  import { calculatePosition, Visibility } from "ui/src/tooltip";
+  export let value: string | undefined = undefined;
+  export let position: Position = "right";
 
-  export let value = "";
-
-  export let position: CSSPosition = CSSPosition.Right;
+  type Position = "top" | "right" | "bottom" | "left";
 
   let container: Element | null = null;
   let message: Element | null = null;
-  let tooltip = { className: Visibility.Hidden, x: 0, y: 0 };
+  let tooltip: { className: "hidden" | "visible"; x: number; y: number } = {
+    className: "hidden",
```

Comment:
Can we make `className` a separate variable? It’s always set separately

Reply:
3125c96ba5e44319c27d4951f41efb23b3902da7